fili-core/src/test/groovy/com/yahoo/bard/webservice/util/UtilsSpec.groovy

```diff
@@ -0,0 +1,106 @@
+// Copyright 2017 Yahoo Inc.
+// Licensed under the terms of the Apache license. Please see LICENSE.md file distributed with this work for terms.
+package com.yahoo.bard.webservice.util
+
+import org.apache.commons.io.FileUtils
+
+import spock.lang.Specification
+
+import java.nio.file.Files
+import java.nio.file.Path
+
+class UtilsSpec extends Specification {
+
+    String fromDir
+    String toDir
+
+    Path fromPath
+    Path toPath
+
+    Path file1
+    Path file2
+    Path file3
+    Path subDir
+    Path file4
+
+    void setup() {
+        fromDir = "target/tmp/dimensionCache/animal/new_lucene_indexes"
+        toDir = "target/tmp/dimensionCache/animal/lucene_indexes"
+
+        fromPath = Files.createDirectory(new File(fromDir).getAbsoluteFile().toPath())
+        toPath = Files.createDirectory(new File(toDir).getAbsoluteFile().toPath())
+
+        file1 = fromPath.resolve("segments_1")
+        file2 = fromPath.resolve("_1.cfs")
+        file3 = fromPath.resolve("_1.si")
+        subDir = fromPath.resolve("subDir")
+
+        Files.createFile(file1)
+        Files.createFile(file2)
+        Files.createFile(file3)
+        Files.createDirectory(subDir)
+        file4 = subDir.resolve("subDirFile")
+        Files.createFile(file4)
```

Comment:
Then something like 

```groovy
file1 = Files.createTempFile(fromPath, "", "segments_1").toAbsolutePath()
file2 = Files.createTempFile(fromPath, "", "_1.cfs").toAbsolutePath()
file3 = Files.createTempFile(fromPath, "", "_1.si").toAbsolutePath()
subDir = Files.createTempDirectory(fromPath, "subDir").toAbsolutePath()
file4 = Files.createTempFile(subDir, "", "subDirFile").toAbsolutePath()
```

possibly also changing the names from `"segments_1"` to something like `"file1"`